custom/enikshay/two_b_datamigration/management/commands/import_drtb_cases.py

```diff
@@ -1662,6 +1665,20 @@ def clean_socioeconomic_status(value):
         "unknown": "unknown",
     }[value.lower()]
 
+def clean_marital_status(value):
+    if not value:
+        return None
+    clean_value = value.lower().strip()
+    try:
+        return {
+            "unmarried": "unmarried",
+            "married": "married",
+            "widowed": "widowed",
+            "separated": "separated",
+        }[clean_value]
```

Comment:
why do you need to do this? It looks like a 1-1 relationship. 
You could instead have done: 
```py
if clean_value in ["unmarried", "married", "widowed", "separated"]: 
    return clean_value
else:
    raise FieldValidationFailure(value, "Marital Status")
```